<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>webkit</class>
 <widget class="QWidget" name="webkit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>560</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>WebKit Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="kcfg_autoLoadImages">
        <property name="text">
         <string>Autoload images</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="7">
       <widget class="Line" name="line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="kcfg_linksIncludedInFocusChain">
        <property name="text">
         <string>Links included in focus chain</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="kcfg_dnsPrefetch">
        <property name="text">
         <string>Prefetch DNS entries</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="kcfg_zoomTextOnly">
        <property name="text">
         <string>Zoom text only</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="kcfg_javascriptEnabled">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>JavaScript support</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="kcfg_printElementBackgrounds">
        <property name="text">
         <string>Print element backgrounds</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="kcfg_offlineStorageDatabaseEnabled">
        <property name="text">
         <string>Offline storage database</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QCheckBox" name="kcfg_javaEnabled">
        <property name="text">
         <string>Java support</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="kcfg_javascriptCanOpenWindows">
        <property name="text">
         <string>JavaScript can open windows</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="kcfg_offlineWebApplicationCacheEnabled">
        <property name="text">
         <string>Offline web application cache</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="kcfg_javascriptCanAccessClipboard">
        <property name="text">
         <string>JavaScript can access clipboard</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QCheckBox" name="kcfg_localStorageEnabled">
        <property name="text">
         <string>Local Storage</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Plugin Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>When loading web pages:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="kcfg_pluginsEnabled">
        <item>
         <property name="text">
          <string>Autoload Plugins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manually Load Plugins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Never Load Plugins</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>User Style Sheet</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>User CSS path:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KUrlRequester" name="kcfg_userCSS">
        <property name="filter">
         <string>*.css</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
